### ISSUE TITLE
Call for translation in multiple languages

### ISSUE THREAD

**Issue by mishmanners** [mishmanners/mish-friendly-food#10]

**Comment by mishmanners** [comment]
People have also added:
- Danish
- German; requested (now added to list above)

**Comment by mishmanners** [comment]
Need to tweet with the top languages still to add.

**Comment by alt-shreya** [comment]
@mishmanners I could help with translating documents to Hindi. Also German, if that's something you're looking for.

**Comment by mishmanners** [comment]
@alt-shreya yes please. Can you please take a look at the [database file](https://github.com/mishmanners/mish-friendly-food/blob/main/_data/database.json) and check through the German and Hindi translations. Feel free to add extras and let me know if you need anything. Apologises for the slow response on this. I was swamped for Hacktoberfest!

**Comment by mishmanners** [comment]
Actually Hindi has already been done, but I'd love you to do German if you could @alt-shreya 😄

**Comment by mishmanners** [comment]
Ping @alt-shreya on the German.

**Comment by alt-shreya** [comment]
@mishmanners noted, I think the email got lost in my inbox, but will begin work ASAP :)

**Comment by mishmanners** [comment]
Thanks for completing the German @alt-shreya 😄

**Comment by AnsellMaximilian** [comment]
@mishmanners, I'd love to help complete the Indonesian translations if you're still accepting contributions. 😄

**Comment by mishmanners** [comment]
100% @AnsellMaximilian. If you'd like to go through them, double check, change, and complete the Indonesian translations that'd be awesome. Send through a PR and I'll merge it 😄

**Comment by AlexanderAmmann** [comment]
Hello, @mishmanners I would like to help with the German langugage if you are still acceptin contributions.

**Comment by mishmanners** [comment]
Yes please, happy for you to do this and submit a PR
